### PR TITLE
Slt2

### PR DESCRIPTION
**Pull request by f-bayer15**
Created a function to ask the LLM about possible collaboration topics for two researchers. The possible questions can easily be expanded on later.

### CODE REVIEW

File: src/justify_similarities.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "What are common study areas of these researchers?",
+    "What could be possible titles of papers for these researchers to collaborate on?"]
+
+    if (q_index not in range(len(questions)-1)):
+        print("Number given for q_index must be between 0 and " + str(len(questions)-1))
+        return None
+    else:
+        question = questions[q_index] + " Please keep your answer short and concise"
+
+    answer = get_answer(question, text_dir, text_list)
```
Comment (MarioStanke) [inline src/justify_similarities.py:32]: How are the publication lists separated? I believe only by a newline and relying on the name being in the first line and being recognized as such. That is not clear. Some publication titles could start with a name, too, e.g. "Vorname Name, eine Biografie". Please make it clear, e.g.

<Researcher 1>
...


<Researcher 2>
...

**Review by MarioStanke — CHANGES_REQUESTED** [review]
Just inject a few text snippets, see inline comments.

**Comment by f-bayer15** [comment]
made the requested changes and added a seperator list argument to explicitely seperate the chunks from each other. Ready to e reviewed again.

**Review by MarioStanke — APPROVED** [review]
LGTM

**Pull request merged**